homeassistant/components/screenlogic/const.py

```diff
@@ -1,7 +1,16 @@
 """Constants for the ScreenLogic integration."""
+from screenlogicpy.const import COLOR_MODE
+
+from homeassistant.util import slugify
 
 DOMAIN = "screenlogic"
 DEFAULT_SCAN_INTERVAL = 30
 MIN_SCAN_INTERVAL = 10
 
+SERVICE_SET_COLOR_MODE = "set_color_mode"
+ATTR_COLOR_MODE = "color_mode"
+SUPPORTED_COLOR_MODES = {
+    slugify(name): num for num, name in COLOR_MODE.NAME_FOR_NUM.items()
```

Comment:
Why slugify?

Reply:
The source names have capitals and spaces in them. I need to map a name to a number and A) I didn't know if service data supported full strings, and B) I thought it might be simpler for people in yaml mode to not have to worry about the exact punctuation.
Maybe I'm thinking too much in the pre-config-entry/yaml times. Would it be prefered to us the full natural language version of the options?

Reply:
Seems ok to me. Just wanted to make sure since it would be a breaking change if we wanted to adjust it later.